packages/harmony/src/components/completion-check/CompletionCheck.module.css

```diff
@@ -0,0 +1,40 @@
+.container {
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  width: 16px;
+  height: 16px;
+}
+
+.defaultCompletionIcon {
+  position: absolute;
+  display: inline-flex;
+  align-items: center;
+  height: 12px;
+  width: 12px;
+  border: 2px solid var(--harmony-neutral);
+  border-radius: 50%;
+}
+
+.completionIcon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  border-radius: 50%;
+  z-index: 15;
```

Comment:
concerned about global z-indexes. JD showed me this https://css-tricks.com/almanac/properties/i/isolation/
See harmony `TextInput.module.css` for an example